iosApp/iosApp/Pages/NearbyTransit/NearbyViewModel.swift

```diff
@@ -83,17 +85,21 @@ class NearbyViewModel: ObservableObject {
                 self.nearbyState.loading = false
                 self.selectingLocation = false
             }
+            let errorKey = "NearbyViewModel.getNearby"
             switch await callApi({
                 try await nearbyRepository.getNearby(global: global, location: location.coordinateKt) }) {
             case let .ok(result):
+                errorBannerRepository.clearDataError(key: errorKey)
                 if Task.isCancelled { return }
                 nearbyState.nearbyByRouteAndStop = result.data
                 nearbyState.loadedLocation = location
-                nearbyState.error = nil
-            case let .error(error):
+            case .error:
                 withUnsafeCurrentTask { thisTask in
                     if self.fetchNearbyTask?.hashValue == thisTask?.hashValue {
-                        self.nearbyState.error = error.message
+                        errorBannerRepository.setDataError(
+                            key: errorKey,
+                            action: { [self] in getNearby(global: global, location: location) }
```

Comment:
Without it, I get a Swift compile error:
> Call to method 'getNearby' in closure requires explicit use of 'self' to make capture semantics explicit

It offers two solutions - capturing `self` explicitly or calling `self.getNearby` explicitly.

- Capturing `self` explicitly - `{ [self] in getNearby() }` - isn't actually enough because "Variable other than 'self' captured here under the name 'self' does not enable implicit 'self'", although I'm not sure why capturing `self` doesn't capture `self`.
- The suggested fix to that is `{ [self] in self.getNearby() }`, which compiles but which SwiftFormat rewrites back to `{ [self] in getNearby() }`
- Just doing `{ self.getNearby() }` appears to make both Swift and SwiftFormat happy, though.